<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgressBarDialog</class>
 <widget class="QDialog" name="ProgressBarDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>363</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>431</width>
    <height>359</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>431</width>
    <height>363</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Demo dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(100, 100, 100);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>411</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="title">
    <string>Proprties</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Reflection offset</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSlider" name="slider">
      <property name="minimum">
       <number>-15</number>
      </property>
      <property name="maximum">
       <number>15</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="reflectionsCheckBox">
      <property name="styleSheet">
       <string notr="true">QCheckBox{
border: 2px black;
}

QCheckBox::indicator {

 }</string>
      </property>
      <property name="text">
       <string>Reflections</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Activity name:</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background: white; border: 2px solid black;border-radius: 5px;
color:black;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="title">
    <string>Control</string>
   </property>
   <widget class="ActivityWidget" name="activityWidget" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ActivityWidget</class>
   <extends>QWidget</extends>
   <header>ActivityWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
